docs/docsource/reference/blobserver/Account/ListContainers.rst

```diff
@@ -0,0 +1,377 @@
+
+
+Response Body
+~~~~~~~~~~~~~
+
+The format of the response body is as follows.
+
+   ::
+
+      <?xml version="1.0" encoding="utf-8"?>  
+      <EnumerationResults ServiceEndpoint="https://myaccount.blob.example.com">  
+        <Prefix>string-value</Prefix>  
+        <Marker>string-value</Marker>  
+        <MaxResults>int-value</MaxResults>  
+        <Containers>  
+          <Container>  
+            <Name>container-name</Name>  
+            <Properties>  
+              <Last-Modified>date/time-value</Last-Modified>  
+              <Etag>etag</Etag>  
+              <LeaseStatus>locked | unlocked</LeaseStatus>  
+              <LeaseState>available | leased | expired | breaking | broken</LeaseState>  
+              <LeaseDuration>infinite | fixed</LeaseDuration> 
+              <PublicAccess>container | blob</PublicAccess>
+              <HasImmutabilityPolicy>true | false</HasImmutabilityPolicy>
+              <HasLegalHold>true | false</HasLegalHold>
+            </Properties>  
+            <Metadata>  
+              <metadata-name>value</metadata-name>  
+            </Metadata>  
+          </Container>  
+        </Containers>  
+        <NextMarker>marker-value</NextMarker>  
+      </EnumerationResults>  
+
+The ``Prefix``, ``Marker``, and ``MaxResults`` elements are only present if they
+were specified on the URI. The ``NextMarker`` element has a value only if the
+list results are not complete.
+
+The ``Metadata`` element is present only if the ``include=metadata`` parameter
+was specified on the URI. Within the ``Metadata`` element, the value of each
+name-value pair is listed within an element corresponding to the pair's name.
+
+If a metadata name-value pair violates the naming restrictions enforced by the
+2009-09-19 version, the response body indicates the problematic name within an
+``x-ms-invalid-name`` element, as shown in the following XML fragment:
+
+   ::
+
+        
+      <Metadata>  
+        <MyMetadata1>first value</MyMetadata1>  
+        <MyMetadata2>second value</MyMetadata2>  
+        <x-ms-invalid-name>invalid-metadata-name</x-ms-invalid-name>  
+      </Metadata>  
+        
+Container public permissions are provided in the PublicAccess property. It
+indicates whether data in the container may be accessed publicly and the level
+of access. Possible values include:
+
+- container: Indicates full public read access for container and blob
+  data. Clients can enumerate blobs within the container via anonymous request,
+  but cannot enumerate containers within the storage account.
+
+- blob: Indicates public read access for blobs. Blob data within this container
+  can be read via anonymous request, but container data is not
+  available. Clients cannot enumerate blobs within the container via anonymous
+  request.
+
+If this property is not specified in the section, the container is private to
+the account owner.
+
+``HasImmutabilityPolicy`` is ``true`` if the container has an
```

Comment:
We don't support  `HasImmutabilityPolicy` and `HasLegalHold` so they will always be false

Reply:
removed/clarified all instances. Changed value to false in code samples.